.github/workflows/bahmni_lite_docker_env.yaml

```diff
@@ -42,9 +42,9 @@ jobs:
       - name: Create/Update Containers
         run: |
           if [ "$UPDATE_BAHMNI_OPTION" == "update_without_removing_volumes" ]; then
-            command="docker compose up -d"
+            command="docker compose --env-file .env.dev up -d"
           elif [ "$UPDATE_BAHMNI_OPTION" == "update_and_remove_volumes" ]; then
-            command="docker compose down -v && docker compose up -d"
+            command="docker compose down -v && docker compose --env-file .env.dev up -d"
```

Comment:
--env-file option should be added for down command as well

Reply:
Also add --env-file option at line 63 as well for the docker compose pull command

Reply:
sure Mohan, will add it